lib/actions/model.js

```diff
@@ -61,4 +61,26 @@ module.exports = {
       return xhr;
     };
   },
+
+  validate(brainstemKey, modelId, attributes, options = {}) {
+    const {
+      validateOptions = {},
+      preValidateAction,
+      postValidateAction,
+      trackKey,
+    } = options;
+
+    const storageManager = StorageManager.get();
+
+    return (dispatch) => {
+      if (preValidateAction) dispatch(preValidateAction);
+
+      const Model = storageManager.storage(brainstemKey).model;
+      const validationErrors = new Model({ id: modelId }).validate(attributes, validateOptions);
```

Comment:
More like you're just using the shell of a model and invoking another model's method. Brainstem wouldn't return an instance because this would be for a new model.

Creating a `new` instance is fine, but I hope it's understood what my approach was accomplishing.